lib/simplecov-html/version.rb

```diff
@@ -1,7 +1,7 @@
 module SimpleCov
   module Formatter
     class HTMLFormatter
-      VERSION = "0.10.1"
+      VERSION = "0.10.1".dup
 
       def VERSION.to_a
```

Comment:
I'm leaning towards VERSION just being a standard string, and removing all the helper methods. They're not used anywhere in the code from what I could see, so the only downside is it might break things in other libraries… the odds of that, I would hope, are pretty small.